src/GLdispatch/vnd-glapi/entry_files.mk

```diff
@@ -3,35 +3,35 @@
 # The conditionals here are all set from the configure script.
 
 if GLDISPATCH_TYPE_X86_TLS
-MAPI_GLDISPATCH_ENTRY_FILES = $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_tls.c
+MAPI_GLDISPATCH_ENTRY_FILES = mapi/entry_x86_tls.c
 endif
 
 if GLDISPATCH_TYPE_X86_TSD
-MAPI_GLDISPATCH_ENTRY_FILES = $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_tsd.c
-MAPI_GLDISPATCH_ENTRY_FILES += $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_64_common.c
-MAPI_GLDISPATCH_ENTRY_FILES += $(top_srcdir)/$(MAPI_PREFIX)/entry_common.c
+MAPI_GLDISPATCH_ENTRY_FILES = mapi/entry_x86_tsd.c
+MAPI_GLDISPATCH_ENTRY_FILES += mapi/entry_x86_64_common.c
+MAPI_GLDISPATCH_ENTRY_FILES += mapi/entry_common.c
 endif
 
 if GLDISPATCH_TYPE_X86_64_TLS
-MAPI_GLDISPATCH_ENTRY_FILES = $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_64_tls.c
-MAPI_GLDISPATCH_ENTRY_FILES += $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_64_common.c
-MAPI_GLDISPATCH_ENTRY_FILES += $(top_srcdir)/$(MAPI_PREFIX)/entry_common.c
+MAPI_GLDISPATCH_ENTRY_FILES = mapi/entry_x86_64_tls.c
+MAPI_GLDISPATCH_ENTRY_FILES += mapi/entry_x86_64_common.c
+MAPI_GLDISPATCH_ENTRY_FILES += mapi/entry_common.c
```

Comment:
Adding the $(srcdir) prefix breaks it just like using $(top_srcdir) would.

It looks like "make distcheck" only fails if you've already run a normal "make", so something from the normal build must be interfering with the distcheck build.

And, it looks like mapi/libglapi_la-entry_x86_64_tls.lo isn't getting built at all in the distcheck build. Maybe some sort of dependency problem?